bindata/cloud-network-config-controller/managed/controller.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
     release.openshift.io/version: "{{.ReleaseVersion}}"
     networkoperator.openshift.io/non-critical: ""
     network.operator.openshift.io/cluster-name:  {{.ManagementClusterName}}
+  labels:
+    # used by PodAffinity to prefer co-locating pods that belong to the same hosted cluster.
+    hypershift.openshift.io/hosted-control-plane: {{.HostedClusterNamespace}}
```

Comment:
Is this a standard hypershift label? Did not see it in https://hypershift-docs.netlify.app/how-to/distribute-hosted-cluster-workloads/.

Reply:
it is a label that the control-plane-operator uses: https://github.com/openshift/hypershift/blob/main/support/config/deployment.go#L27